pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -111,7 +112,26 @@ const (
 )
 
 var (
-	supportedLanguages = []language{java, js, python, dotnet, ruby}
+	supportedLanguages = []language{
+		java,
+		js,
+		python,
+		dotnet,
+		ruby,
```

Comment:
N.B. Added php as the minimum version is _specified_ but it is not yet a supported language.